_dashboards/dashboard/index.md

```diff
@@ -0,0 +1,126 @@
+
+## Defining terminology
+
+Learn some useful terminology for working with OpenSearch Dashboards and the **Dashboard** application.
+
+- _Dashboards_ is the abbreviated proper name for OpenSearch Dashboards. OpenSearch Dashboards is an open-source visualization tool designed to work with OpenSearch.
+- _Dashboard_ is the application in OpenSearch Dashboards that is used to track, analyze, and display data at a glance.
+- _dashboard_ or _dashboards_ are common names for a tool used to visually display data.
+- _Panel_ is a term used to refer to a visualization displayed on a dashboard. The terms _panel_ and _visualization_ may be used interchangeably throughout this and other Dashboard-related documentation.
+
+The following tutorial assumes you're either using your existing installation of OpenSearch Dashboards or using the [OpenSearch playground](https://playground.opensearch.org/app/home#/). Depending on which one you use, certain capabilities aren't available. For example, sample datasets may not be in your existing installation and saving a dashboard isn't an option in the playground.
+{: .note}
+
+## Creating a dashboard and adding an existing visualization
+
+To create the dashboard and add the sample visualization:
+
+1. Connect to `https://localhost:5601`. The username and password are `admin`. Alternatively, connect to your local environment or the [OpenSearch playground](https://playground.opensearch.org/app/home#/).
+1. From the navigation panel, choose **Dashboard**.
+1. From the **Dashboards** window, choose **Create Dashboard**.
+1. Select the calendar icon and set the time filter to **Last 30 days**.
+1. From the panel, choose **Add an existing**.
+1. From the **Add panels** window, choose **[eCommerce] Promotion Tracking**, and then select `x` to close the panel.
+
+You've now created the following basic dashboard with a single panel, which you'll continue using throughout this tutorial.
+
+![Basic dashboard with single panel]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-basic.png)
+
+## Creating visualizations
+
+Continuing with the dashboard you created in the preceding steps, you'll create a new visualization and save it to the dashboard.
+
+1. From the dashboard toolbar, choose **Create new**, then **Visualization**.
+1. From the **New Visualization** window, choose **Vertical Bar** and then select the index pattern **opensearch_dashboards_sample_data_ecommerce**. 
+1. From the toolbar, select **Save**
+1. In the **Save visualization** window, add a title for the visualization. For example, the title for the gauge chart panel is [eCommerce] Orders.
+1.  Select **Save and return**.  
+
+The gauge chart visualization is now saved and you are taken back to the dashboard. You'll see two visualizations on the dashboard, like the following: 
+
+![Dashboard showing visualizations combined in a single view]({{site.url}}{{site.baseurl}}/images/dashboards/dashboard-combined.png)
```

Comment:
These steps and the screenshot refer to a gauge chart, but L62 specifies "Vertical Bar", and should be updated to match.